Makefile

```diff
@@ -47,3 +47,8 @@ pre-commit:
 	# setup pre commit dependencies
 	pip3 install -r requirements/integration.txt
 	pre-commit install
+
+prettier:
```

Comment:
What do you think about making it possible to run just one or the other in addition to running them all? If working on python, it might be nice to be able to run only black without also running prettier and vis-versa.

Reply:
@benjreinhart i agree i'll split the commands and have `make py-prettier` and `make js-prettier`

Reply:
can we call it `prettier` and `black` so it's clear which tool is actually running? Or `js-format` and `py-format`? Prettier supports various file types, and there's a [python plugin](https://github.com/prettier/plugin-python), so it may be a bit confusing which tool is running and needs to be installed. Trying to avoid any "py-prettier is failing even though I have prettier installed" issues.